README.md

```diff
@@ -886,10 +886,24 @@ require "google/cloud/vision"
 
 vision = Google::Cloud::Vision.new
 
-image = vision.image "path/to/landmark.jpg"
-
-landmark = image.landmark
-landmark.description #=> "Mount Rushmore"
+gcs_image_uri = "gs://gapic-toolkit/President_Barack_Obama.jpg"
```

Comment:
The README for the other GAPIC-only gems is generated. Should the README also be changed out for the generated file?